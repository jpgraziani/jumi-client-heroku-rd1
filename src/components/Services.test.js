import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Services from './Services'

describe(`Services component`, () => {
  it('renders a .Services by default', () => {
    const wrapper = shallow(<Services />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})