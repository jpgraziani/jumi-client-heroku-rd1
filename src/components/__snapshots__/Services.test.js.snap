// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component renders a .Hero by default 1`] = `<header />`;

exports[`Services component renders a .Services by default 1`] = `
<section
  className="services"
>
  <Title
    title="about"
  />
  <div
    className="services-center"
  >
    <article
      className="service"
      key="0"
    >
      <span>
        <FaBabyCarriage />
      </span>
      <h6>
        baby food/toddler
      </h6>
      <p>
        We are currently exploring new foods for our child so we are always updating our site with new ideas.
      </p>
    </article>
    <article
      className="service"
      key="1"
    >
      <span>
        <FaHandHoldingHeart />
      </span>
      <h6>
        healthy lifestyle
      </h6>
      <p>
        We beleive in eating well and exercise can help give a happy healthy lifestyle. Look good, feel good.
      </p>
    </article>
    <article
      className="service"
      key="2"
    >
      <span>
        <FaFish />
      </span>
      <h6>
        recipes we make daily
      </h6>
      <p>
        We are a new family learning the way around the kitchen, so we make sure our recipes are easy peasy to cook.
      </p>
    </article>
    <article
      className="service"
      key="3"
    >
      <span>
        <FaBook />
      </span>
      <h6>
        education
      </h6>
      <p>
        We believe a continued healthy lifestyle comes from knowledge so we are here to share our findings on this journey.
      </p>
    </article>
  </div>
</section>
`;
